### PR TITLE
feat: add inactive fair state to Fair2

### PR DESCRIPTION
**Pull request by anandaroop**
The type of this PR is: **Feature**

This PR resolves [FX-2347]

### Description

To be consistent with the current Fair view this PR updates the Fair2 view to be pared down for inactive fairs:

Also updates handling for missing images (which maybe shouldn't happen, but I was able to navigate to this one from City Guide, e.g.)

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[FX-2347]: https://artsyproduct.atlassian.net/browse/FX-2347

### CODE REVIEW

File: src/lib/Scenes/Fair2/Components/Fair2Header.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const Fair2Header: React.FC<Fair2HeaderProps> = ({ fair }) => {
 
   return (
     <Box>
-      {!!image && (
+      {image ? (
```
Comment (sweir27) [inline src/lib/Scenes/Fair2/Components/Fair2Header.tsx:53]: Should we keep the `!!`?

File: src/lib/Scenes/Fair2/Components/Fair2Header.tsx [hunk 2]
```diff
@@ -89,6 +91,8 @@ export const Fair2Header: React.FC<Fair2HeaderProps> = ({ fair }) => {
   )
 }
 
+const SafeTopMargin = () => <Spacer mt={6} pt={2} />
```
[the remark this answers is not in the document]
Comment (sweir27) [reply src/lib/Scenes/Fair2/Components/Fair2Header.tsx:94]: I still feel like one of these values should be:
```
const { safeAreaInsets } = useScreenDimensions()
// safeAreaInsets.top
```

But would be worth getting an MX friend to answer that.

File: src/lib/Scenes/Fair2/Fair2.tsx [hunk 3]
```diff
@@ -56,13 +57,15 @@ export const Fair2: React.FC<Fair2Props> = ({ fair }) => {
...
-  ]
+  const sections = isActive
+    ? [
+        "fairHeader",
+        ...(hasArticles ? ["fairEditorial"] : []),
+        ...(hasCollections ? ["fairCollections"] : []),
+        ...(hasFollowedArtistArtworks ? ["fairFollowedArtistsRail"] : []),
+        ...(hasArtworks && hasExhibitors ? ["fairTabs", "fairTabChildContent"] : []),
+      ]
+    : ["fairHeader", "notActive"]
```
Comment (sweir27) [inline src/lib/Scenes/Fair2/Fair2.tsx:68]: Based on the description in the ticket, I believe we also want to include the `fairEditorial` section if it has content + the fair is not yet open.

File: src/lib/Scenes/Fair2/Components/Fair2Header.tsx (hunk 1 above)
Comment (anandaroop) [reply src/lib/Scenes/Fair2/Components/Fair2Header.tsx:53]: `image` is already truthy if present? But I don't feel strongly, can add back (I don't think this was intentional 😃 )

File: src/lib/Scenes/Fair2/Fair2.tsx (hunk 3 above)
Comment (anandaroop) [reply src/lib/Scenes/Fair2/Fair2.tsx:68]: Ah good catch about editorial, lemme update...

Comment (anandaroop) [reply src/lib/Scenes/Fair2/Fair2.tsx]: Updated to include Editorial, e.g.

**Comment by sweir27** [comment]
We can follow up with the top-padding/margin question if we want! I don't think it blocks this PR.

**Pull request merged**